params.go

```diff
@@ -30,6 +32,8 @@ func (c *commandParams) Read(args []string) bool {
 	fs.BoolVar(&c.debug, "debug", false, "enable debug logging for failed tests")
 	fs.BoolVar(&c.debugAll, "debug-all", false, "enable debug logging for all tests")
 	fs.StringVar(&c.jUnitFile, "junit", "", "write JUnit XML output to the specified path")
+	fs.StringVar(&c.genSuppressions, "gen-suppressions", "", "output suppression file")
+	fs.StringVar(&c.suppressions, "suppressions", "", "input suppression file")
```

Comment:
These parameter names and descriptions are a bit vague; if I just saw this list and hadn't read the code, I would have no idea what they did (especially since the PR doesn't include any additions to the command-line instructions in `docs/`).

How about something more specific to what they are doing, such as:

`-list-failed-tests-to` ... `writes names of failed tests to the specified file, one per line`
`-skip-all-from` ... `skips tests named in the specified file, one per line`